modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/jaxrs/exception/mapper/UserAccountFirstNameMustNotExceedMaximumLengthExceptionMapper.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.admin.user.internal.jaxrs.exception.mapper;
+
+import com.liferay.portal.kernel.exception.ContactNameException;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Michele Vigilante
+ */
+@Component(
+	property = {
+		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Headless.Admin.User)",
+		"osgi.jaxrs.extension=true",
+		"osgi.jaxrs.name=Liferay.Headless.Admin.User.UserAccountFirstNameMustNotExceedMaximumLengthExceptionMapper"
+	},
+	service = ExceptionMapper.class
+)
+public class UserAccountFirstNameMustNotExceedMaximumLengthExceptionMapper
+	extends BaseExceptionMapper
+		<ContactNameException.MustNotExceedMaximumLength> {
+
+	@Override
+	protected Problem getProblem(
+		ContactNameException.MustNotExceedMaximumLength
+			mustNotExceedMaximumLength) {
+
+		return new Problem(
+			Response.Status.BAD_REQUEST, "The user account name is invalid");
```

Comment:
it's not name but given name? (what was in the previous one?) because this should match the attribute name in the yaml